clouddriver-ecs/src/integration/resources/testoperations/createServerGroup-artifact-EC2-TGMappings-multipleLBsAndContainers.json

```diff
@@ -0,0 +1,57 @@
+{
+  "account": "ecs-account",
+  "application": "ecs",
+  "availabilityZones": {
+    "us-west-2": [
+      "us-west-2a",
+      "us-west-2c"
+    ]
+  },
+  "capacity": {
+    "desired": 1,
+    "max": 1,
+    "min": 1
+  },
+  "cloudProvider": "ecs",
+  "computeUnits": 256,
+  "credentials": "ecs-account",
+  "dockerImageAddress": "nginx",
+  "ecsClusterName": "integArtifactEC2TgMappingskWithMultipleLBsAndContainers-cluster",
+  "launchType": "EC2",
+  "networkMode": "bridge",
+  "placementStrategySequence": [],
+  "reservedMemory": 512,
+  "stack": "integArtifactsEC2TgMappingsStackWithMultipleLBsAndContainers",
+  "freeFormDetails" : "detailTest",
+  "targetGroupMappings": [
+    {
+      "containerName": "application1",
+      "containerPort": 80,
+      "targetGroup": "integArtifactEC2TgMappings-targetGroupForPort80"
+    },
+    {
+      "containerName": "application2",
+      "containerPort": 84,
+      "targetGroup": "integArtifactEC2TgMappings-targetGroupForPort84"
+    }
+  ],
+  "useTaskDefinitionArtifact" : true,
+  "taskDefinitionArtifactAccount"  :  "my-github",
+  "resolvedTaskDefinitionArtifact": {
+    "account": "ecs-account",
+    "type": "ecs",
+    "customKind" :  true,
+    "name" : "applications",
+    "location" : "us-west-2",
+    "reference" : "refernce",
+    "metadata" : null,
+    "artifactAccount" : "my-github",
+    "provenance" : "prov",
+    "uuid" : "uid-123"
+  },
+  "containerToImageMap":
+  {
+    "application1": "PLACEHOLDER",
+    "application2": "PLACEHOLDER"
```

Comment:
it's not helpful to keep these values as "PLACEHOLDER" since:
1) they're the same, so we can't tell if they've been mis-mapped to the wrong container definitions when we assert
2) this is the same text which typically appears in the task def artifact (I see you removed it from the artifact used for this test, but the behavior we want to test is that this "PLACEHOLDER" is _overwritten_ by the correct image ID). 

Please update to "app1/image" / "app2/image" or something similar.

Reply:
Aahhh, okay I understood now.